docs/changelog.rst

```diff
@@ -11,6 +11,18 @@ with advance notice in the **Deprecations** section of releases.
 
 .. towncrier release notes start
 
+v3.14.6 (2020-03-20)
+--------------------
```

Comment:
This file should not be touched, rather individual files into https://github.com/tox-dev/tox/blob/master/docs/changelog/README.rst

Reply:
done :+1: